sdk/identity/src/token_credentials/default_credentials.rs

```diff
@@ -3,14 +3,24 @@ use super::{
 };
 use azure_core::TokenResponse;
 
-#[derive(Debug, Default)]
+#[derive(Debug)]
 /// Provides a mechanism of selectively disabling credentials used for a `DefaultAzureCredential` instance
 pub struct DefaultAzureCredentialBuilder {
     include_environment_credential: bool,
     include_managed_identity_credential: bool,
     include_cli_credential: bool,
 }
 
+impl Default for DefaultAzureCredentialBuilder {
+    fn default() -> Self {
+        Self {
+            include_environment_credential: true,
+            include_managed_identity_credential: true,
+            include_cli_credential: true,
+        }
+    }
```

Comment:
```suggestion
        Self::new()
```

Reply:
I'm not sure what this suggested change would accomplish. Self::new() calls Self::default(), so how would calling Self::new() from the implementation of Default even work?

Reply:
I missed that.  Sorry.